web/components/home/Home.js

```diff
@@ -176,7 +177,7 @@ class Home extends React.Component {
                         className="scale-width-mobile"
                       />
                     </Link>
-                    {/**<a
+                    {/** <a
```

Comment:
Sorry about this haha, I pushed the "comment out mentor button" commit directly to master skipping the whole lint check. 🙉

Reply:
haha no worries